packages/wallet/src/services/BalanceTracker.ts

```diff
@@ -4,6 +4,11 @@ import { TrackerSubject } from './util';
 import { TransactionalObservables } from '..';
 import { combineLatest, map } from 'rxjs';
 
+// TODO: subtract deposit quantity from total utxo coin as it can't be spent
+// Review: not sure how to represent this. Is it 'balance.available$'? 'balance.total$'? a new one?
```

Comment:
How about representing `balance.deposited` explicitly? That will best communicate these allocated funds using domain terminology